source/cloud-native-platform/common-pipeline/publishing-libraries/java.html.md.erb

```diff
@@ -8,9 +8,71 @@ hide_in_navigation: true
 
 # <%= current_page.data.title %>
 
-For publishing Java libraries, we use JitPack as our artifact repository and GitHub Actions to publish the libraries.
+For publishing Java libraries, we now use Azure Artifacts (preferred platform default). Previously, we used JitPack as our artifact repository to publish the libraries, which is now deprecated and scheduled for removal from the platform.
 
-## Publishing with JitPack
+
+## Publishing with Azure DevOps Artifacts
+You will need to set up a Github Action in your repo, using a template as below as a starting point.
+This template will watch for new tags, and upon a new tag being available, will package and publish your project to our Azure DevOps Artifacts feed [hmcts-lib](https://dev.azure.com/hmcts/Artifacts/_artifacts/feed/hmcts-lib).
+
+## Using the Github Action
+
+1. Go to your repository within GitHub
+2. Select the Actions tab at the top
+3. Select 'New workflow'
+4. Scroll down and select 'Trigger ADO Artifact Build' pressing the 'Set up this workflow' button
```

Comment:
this maybe covers above?

Reply:
Yep, github action is available as central template in the org, so following these steps will sort it